.github/workflows/main.yml

```diff
@@ -34,4 +35,4 @@ jobs:
           TRANSIT_DEV_BACKEND_URL: ${{ secrets.TRANSIT_DEV_BACKEND_URL }}
           UPLIFT_BACKEND_URL: ${{ secrets.UPLIFT_BACKEND_URL }}
           VOLUME_BACKEND_URL: ${{ secrets.VOLUME_BACKEND_URL }}
-        run: source venv/bin/activate; python src/main.py
+        run: source venv/bin/activate; python src/main.py --use-config
```

Comment:
--use-config?

Reply:
I changed the running arguments in the program